src/routes.ts

```diff
@@ -19,8 +19,8 @@ export const withdrawAsset = (accountID: string) => `/account/${accountID}/withd
 export const showTransaction = (accountID: string, transactionHash: string) =>
   `/account/${accountID}/tx/${transactionHash}`
 
-export const tradeAsset = (accountID: string, method?: "buy" | "sell") => {
-  return method ? `/account/${accountID}/trade/${method}` : `/account/${accountID}/trade`
+export const tradeAsset = (accountID: string, method?: "buy" | "sell", preselectedAsset?: string) => {
+  return `/account/${accountID}/trade` + (method ? `/${method}` : "") + (preselectedAsset ? `/${preselectedAsset}` : "")
```

Comment:
How about this?
```suggestion
  return [
    `/account/${accountID}/trade`,
    method || "",
    preselectedAsset || ""
  ].filter(fragment => !!fragment).join("/")
```

Reply:
Definitely looks more readable than the string concatenation I used. 👍